android-agent/src/main/kotlin/io/opentelemetry/android/agent/dsl/instrumentation/InstrumentationConfiguration.kt

```diff
@@ -35,26 +38,44 @@ class InstrumentationConfiguration internal constructor(
         )
     }
 
+    /**
+     * Configures activity lifecycle instrumentation.
+     */
     fun activity(configure: ActivityLifecycleConfiguration.() -> Unit) {
         activity.configure()
     }
 
+    /**
+     * Configures fragment lifecycle instrumentation.
+     */
     fun fragment(configure: FragmentLifecycleConfiguration.() -> Unit) {
         fragment.configure()
     }
 
+    /**
+     * Configures ANR detection instrumentation.
+     */
     fun anrReporter(configure: AnrReporterConfiguration.() -> Unit) {
         anr.configure()
     }
 
+    /**
+     * Configures crash reporting instrumentation.
+     */
     fun crashReporter(configure: CrashReporterConfiguration.() -> Unit) {
         crash.configure()
     }
 
+    /**
+     * Configures network requests instrumentation.
```

Comment:
```suggestion
     * Configures network change instrumentation.
```

That one is about monitoring for network changes and not about network requests.